scripts/deployments/RunWorkflows.ps1

```diff
@@ -255,8 +259,8 @@ if ($DeployHubNetworkWithNVA) {
     -SubscriptionId $Context.Variables['var-hubnetwork-subscriptionId'] `
     -ConfigurationFilePath "$($Context.NetworkingDirectory)/$($Context.Variables['var-hubnetwork-nva-configurationFileName'])" `
     -LogAnalyticsWorkspaceResourceId $LoggingConfiguration.LogAnalyticsWorkspaceResourceId `
-    -NvaUsername $NvaUsername `
-    -NvaPassword $NvaPassword
+    -NvaUsername (ConvertFrom-SecureString -SecureString $NvaUsername -AsPlainText) `
```

Comment:
Can the value be passed in as `SecureString` to the function and kept as secure string until it is used?